src/fmu/dataio/datastructure/meta/meta.py

```diff
@@ -114,18 +114,28 @@ class User(BaseModel):
     )
 
 
-class FMUCase(BaseModel):
+class Case(BaseModel):
```

Comment:
I removed some of these prefix `FMU` on the class names because I felt it should be clear that we are coming from FMU